torch/_torch_docs.py

```diff
@@ -6656,6 +6656,30 @@ def merge_dicts(*dicts):
     tensor([ -24.8863, -771.4742])
 """.format(**common_args))
 
+add_docstr(torch.positive,
```

Comment:
Also add an entry for the method to _tensor_docs.py:

https://github.com/pytorch/pytorch/blob/2236f43da0e1f000ea93271daa882116b7cafd90/torch/_tensor_docs.py#L2664